extras/04_windows_vm.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/bash
+
+set -eux
+
+figlet "Deploying windows vm" | lolcat
+
+
+WINDOWS_IMAGE="${WINDOWS_IMAGE:-http://10.8.120.188/Summit2019.raw}"
+VM_SIZE="${VM_SIZE:-60}"
+VM_MEMORY="${VM_MEMORY:-8192}"
+
+SERVER_IP=$( ip -f inet addr show baremetal |  grep inet | awk '/inet / {print $2}' | cut -d/ -f1)
+SERVERS=$(oc get nodes -o jsonpath={.items[*].status.addresses[?\(@.type==\"InternalIP\"\)].address} | xargs)
+for server in $SERVERS; do
+   ssh core@$server "sudo bash -c 'curl -L https://gist.github.com/karmab/8e8ff7f56499822231505fd2f73107e4/raw/771333117255fb9cb5ef161c3aeff1d5e0a6203f/cnv-bridge > /var/lib/cni/bin/cnv-bridge' ; sudo chmod u+x /var/lib/cni/bin/cnv-bridge"
```

Comment:
I'm not sure about this curl, it didn't worked in my deployment.

Reply:
It assumes you've connected to the machines before.  I had to approve the host key for each machine before it would work.

Reply:
addressed using ssh -o StrictHostKeyChecking=no instead